### PR TITLE
consolidate postgres and kafka scripts

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Reference: https://github.com/stolostron/backlog/issues/26694, https://github.com/stolostron/backlog/issues/26674

### CODE REVIEW

File: operator/config/samples/storage/deploy_postgres.sh [hunk 1]
```diff
@@ -1,49 +1,37 @@
 #!/bin/bash
 KUBECONFIG=${1:-$KUBECONFIG}
-currentDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-TARGET_NAMESPACE=${TARGET_NAMESPACE:-"open-cluster-management"}
+currentDir="$( cd "$( dirname "$0" )" && pwd )"
```
Comment (morvencao) [inline operator/config/samples/storage/deploy_postgres.sh:4]: [$0 and BASH_SOURCE](https://stackoverflow.com/questions/35006457/choosing-between-0-and-bash-source) I would recommend to use `BASH_SOURCE[0]`

Comment (yanmxa) [reply operator/config/samples/storage/deploy_postgres.sh]: Thanks! I have replace the `$0` with `BASH_SOURCE[0]`

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**